src/test/java/edu/cornell/kfs/sys/web/mock/ForceUTF8ResponseInterceptor.java

```diff
@@ -0,0 +1,19 @@
+package edu.cornell.kfs.sys.web.mock;
+
+import java.nio.charset.StandardCharsets;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.springframework.web.servlet.HandlerInterceptor;
+import org.springframework.web.servlet.ModelAndView;
+
+public class ForceUTF8ResponseInterceptor implements HandlerInterceptor {
```

Comment:
similar thought about naming this class to denote it being for tests

Reply:
I'll rename this to clarify that it's a testing class.